<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tuneChangeConfigDialog</class>
 <widget class="QDialog" name="tuneChangeConfigDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Dialog</string>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="statusTip">
   <string notr="true"/>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="accessibleName">
   <string notr="true"/>
  </property>
  <property name="accessibleDescription">
   <string notr="true"/>
  </property>
  <property name="windowFilePath">
   <string notr="true"/>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>311</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="usesScrollButtons">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="fileTab">
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <attribute name="title">
     <string notr="true">File</string>
    </attribute>
    <widget class="QDialogButtonBox" name="buttonBoxFile">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>240</y>
       <width>261</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDelFiile">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>240</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true">Delete</string>
     </property>
     <property name="shortcut">
      <string notr="true"/>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTuneName">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="placeholderText">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>LH Channel</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxLHch">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>43</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>128</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxRHch">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>170</y>
       <width>43</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>128</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>170</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>RH Channel</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>57</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Division</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxDiv">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>301</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="currentText">
      <string notr="true"/>
     </property>
     <property name="placeholderText">
      <string notr="true"/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="DivisionTab">
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <attribute name="title">
     <string notr="true">Parts</string>
    </attribute>
    <widget class="QListWidget" name="listWidgetParts">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>381</width>
       <height>211</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDialogButtonBox" name="buttonBoxParts">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>240</y>
       <width>181</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDelDiv">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>240</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true">Delete</string>
     </property>
     <property name="shortcut">
      <string notr="true"/>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <zorder>pushButtonDelDiv</zorder>
    <zorder>listWidgetParts</zorder>
    <zorder>buttonBoxParts</zorder>
   </widget>
   <widget class="QWidget" name="StaffTab">
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <attribute name="title">
     <string notr="true">Staff</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>57</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Top</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Step</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxStaffTop">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxStaffStep">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
    <widget class="QDialogButtonBox" name="buttonBoxStaff">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>240</y>
       <width>261</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonStaff">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>240</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true">Default</string>
     </property>
     <property name="shortcut">
      <string notr="true"/>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
